src/matrixroomhandler.ts

```diff
@@ -12,14 +13,35 @@ import { DiscordBridgeConfig } from "./config";
 import * as Discord from "discord.js";
 import * as log from "npmlog";
 import * as Bluebird from "bluebird";
+import { Permissions } from "discord.js";
 
 const ICON_URL = "https://matrix.org/_matrix/media/r0/download/matrix.org/mlxoESwIsTbJrfXyAAogrNxA";
 const JOIN_DELAY = 6000;
 const HTTP_UNSUPPORTED = 501;
 const ROOM_NAME_PARTS = 2;
 const AGE_LIMIT = 900000; // 15 * 60 * 1000
+const RETRY_INVITE_ACCEPT_DELAY = 5000; // ms
 
 export class MatrixRoomHandler {
+
+  public static GetBotLink(config: any): string {
```

Comment:
Nit: I'm no fan of statics appearing above the properties and constructor. Could it be placed *after* instance methods?

Reply:
This is what tslint made me do :p

I can update the rule too if desired. This method also feels like it's in the wrong class anyways.

Reply:
Yeah, feels like a utility to me.